hack/init-codegen.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
```

Comment:
If this is refereeced from some other place, please comment it and keep the original copyright

Reply:
Not from some other places, like most other scripts, do not need to add copyright